roles/confluent.kafka_rest/tasks/main.yml

```diff
@@ -148,6 +150,7 @@
     group: "{{kafka_rest_group}}"
   notify: restart kafka-rest
   when: not kafka_rest_secrets_protection_enabled|bool
+  diff: "{{ not mask_secrets|bool }}"
```

Comment:
to me if you are using diff, this is a big one that you would want to see the diff of... maybe we need another variable

Reply:
Added `mask_diff` defaulting to true